<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flowShow</class>
 <widget class="QMainWindow" name="flowShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TxPkt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TxPkt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TxByte</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TxByte">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RxPkt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RxPkt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RxByte">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RxByte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RTxPkt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RtxPkt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RtxByte">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RTXByte</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
